### ISSUE TITLE
Show current reported location

### ISSUE THREAD

**Issue by Sprooty** [NSWSESMembers/availability-poc#92]
Show users current location - perhaps show a label with current suburb with hotlink to a map if a user wants to dig deeper.

**Comment by sdunster** [comment]
I think this in reference to the current update location button in settings. Users will see themselves on a map when they're responding so I don't really see the need for this.

**Comment by Sprooty** [comment]
I think the ability for a user to verify their current location will be more a peace of mind item.

Suspect we could expand on the app eventually to allow for real-time 24x7 tracking - not just during response.

**Comment by sdunster** [comment]
Fair enough but this is a wishlist-level feature imo. GPS is generally pretty trustworthy.
